content/02.body.md

```diff
@@ -1,13 +1,23 @@
 ## Introduction
 
-Networks contain information about relationships between entities ("edges between nodes").
-A node's degree is the number of relationships it has in the network.
+Networks contain information about relationships between entities (referred to here as "edges" between "nodes").
+A node's degree is the number of edges it has in the network.
+`TODO: do we need to discuss in-degree vs out-degree and hetnet degree here?`
 Networks contain many nodes, whose degrees can be aggregated to form the network's degree distribution.
-Because different nodes can have very different degrees, real networks have highly variable degree distributions (Figure {@fig:hetionet}).
+Because different nodes can have very different degrees, real networks generally have highly variable degree distributions [].
+This is especially true for networks encoding biomedical knowledge or assays — where natural forces such as preferential attachment inherent to the problem domain combine with observation-based influences such as study methodology — to create non-uniform degree distributions (Figure {@fig:hetionet}).
 
-![Degree distribution can vary greatly between networks, even between the source and target degree distributions within the same (directed or bipartite) network.
-  Shown are seven degree distributions for six edge type subnetworks of Hetionet [@doi:10.7554/eLife.26726].
-  (The final two are the source and target distributions of the gene-regulates-gene directed edge type.)
+![
```

Comment:
For this figure, have we looked at including the Gene-interacts-Gene edge? Would like to show an undirected edge as a helpful reference.

Reply:
I could include it. Compound-resembles-compound is included and is undirected

Reply:
My bad. Updated in https://github.com/greenelab/xswap-manuscript/pull/43/commits/b5561805ffd1edc43e16151a4306c27f6caa2024

It may be helpful to show both orientations of a bipartite edge.

Also good to use unicode separator for undirected edges. See https://git.dhimmel.com/rephetio-manuscript/#tbl:metaedges

Reply:
I chose the metaedges to illustrate the variability of degree distributions. The final two are source/target for a single metaedge just to show that those can be different too. I'm not opposed to adding more distributions, though distributions that are similar to those already shown may not add much to the figure.